ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -4130,3 +4131,130 @@ export const testCollapseBoxPreview = (): Promise<boolean> =>
       ],
     });
   });
+
+export const testTypography = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      await whisper.create({
+        label: 'Are default values displayed correctly?',
+        onClose: () => {
+          console.debug('closed');
+        },
+        components: [
+          {
+            type: WhisperComponentType.Typography,
+            body: 'default = aligin: "inherit", paragraph: false, variant: "body1"',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            align: Align.Center,
+            body: 'Align: Center',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            align: Align.Justify,
+            body: 'Align: Justify',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            align: Align.Left,
+            body: 'Aligin: Left',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            align: Align.Right,
+            body: 'Aligin: Right',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            tooltip: 'tooltip test',
+            body: 'Tooltip',
+          },
+          {
+            type: WhisperComponentType.Typography,
+            variant: 'body1',
```

Comment:
We usually make options a type instead of using a string directly. Just talked to Harrison, `variant` could be : 
```
'body1'
| 'body2'
| 'button'
| 'caption'
| 'h1'
| 'h2'
| 'h3'
| 'h4'
| 'h5'
| 'h6'
| 'inherit'
| 'overline'
| 'subtitle1'
| 'subtitle2'
| string
```
And then we have to figure what other strings it could be

Reply:
made a type and enum for Variant